apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/r4/providers/pac/R4ClaimResourceProvider.java

```diff
@@ -25,13 +25,9 @@ public class R4ClaimResourceProvider extends AbstractR4ResourceProvider<Claim> {
     return ClaimTypeV2.parse(typeText);
   }
 
-  /**
-   * Returns a set of all supported resource types.
-   *
-   * @return Set of all supported resource types.
-   */
+  /** {@inheritDoc} */
```

Comment:
inheritDoc is the default.  It's not necessary if no implementation specific doc is present.

Reply:
This seems to be getting added to avoid hits from the pipeline validations for javadocs